lib/shortcode/easy-digital-downloads.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+ * Shortcode Template File for Easy Digital Downloads
+ */
+
+
+/**
+ * Create our shortcode for Easy Digital Downloads
+ */
+add_action( 'init', 'sandwich_easy_digital_downloads', 11 );
+function sandwich_easy_digital_downloads() {
+
+	// Check if Shortcake exists
+	if ( ! function_exists( 'shortcode_ui_register_for_shortcode' ) ) {
+		return;
+	}
+
+	// Register Shortcake UI for Easy Digital Downloads Download History
```

Comment:
Add a check whether EDD is installed. We don't have to create the UI for all of these if EDD isn't activated. You can do a `class_exists` on the main EDD class like how the Contact Form 7 shortcode does it.